### ISSUE TITLE
Unraid, el sistema operativo para servidores domésticos | Yayitazale's blog

### ISSUE THREAD

**Issue by yayitazale** [yayitazale/hexo_comments#3]
https://blog.yayitazale.duckdns.org/2021/01/31/Unraid/ 

 Hexo Theme Keep

**Comment by yayitazale** [comment]
a

**Comment by Mifulapirus** [comment]
Qué buena explicación! 
Entonces tú usas un thin client y una VM como ordenador personal?

**Comment by yayitazale** [comment]
> @Mifulapirus
> Qué buena explicación! 
> Entonces tú usas un thin client y una VM como ordenador personal?

No, en mi caso tengo por un lado el servidor y por otro un PC completo ya que para jugar a videojuegos el servidor que tengo montado no es lo suficientemente potente. 

Hay una pruebas hechas por algunos youtubers que montan unos servidores con mejores especificaciones para usar una VM como ordenador principal:

- https://www.youtube.com/watch?v=9syF1aWLtwE
- https://www.youtube.com/watch?v=LuJYMCbIbPk

Los resultados son bastante decentes, pero creo que todavía es complicado la experiencia sea la misma. 

Si en tu uso normal no vas a jugar a videojuegos, es perfectamente factible tener esa configuración. De hecho, con una máquina lo suficientemente potente, Unraid podría ser el servidor principal de una pequeña empresa teniendo los PCs de los trabajadores virtualizados y todos los servicios como VPN, nextcloud, cámaras de vigilancia, algunos servidores dedicados etc.
